app/views/schools/advice/electricity_costs/_analysis.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= render 'tariff_note', school: @school, analysis_dates: @analysis_dates, complete_tariff_coverage: @complete_tariff_coverage, periods_with_missing_tariffs: @periods_with_missing_tariffs %>
+
+<% if @multiple_meters || @analysis_dates.months_of_data > 23 %>
+  <p><%= advice_t('electricity_costs.analysis.summary') %></p>
+<% end %>
+
+<ul>
```

Comment:
maybe move the `<ul>` & `<\ul>` between the `<% if @multiple_meters %>...<% end %>` statement?